src/visitor.rs

```diff
@@ -3,7 +3,7 @@ use std::{collections::HashSet, fmt::Formatter, marker::PhantomData, str::FromSt
 
 use crate::stmt::LogFlag;
 
-/// Deserialize null, a string or string sequence into a Option<Vec<String>>.
+/// Deserialize null, a string or string sequence into an `Option<Vec<String>>`.
```

Comment:
Oxford comma ;)

I guess the type could also be written as a reference.

```suggestion
/// Deserialize null, a string, or string sequence into an [Option<Vec<String>>].
```